app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -84,14 +84,14 @@
         <p id="password-helper-text" class="fs-xs color-base-60 mt-1">Minimum 8 characters</p>
       </div>
 
-      <% needs_owner_secret_field = ENV["FOREM_OWNER_SECRET"].present? && Settings::General.waiting_on_first_user %>
-      <% if needs_owner_secret_field %>
-        <% if params[:owner_secret].present? %>
-          <%= f.hidden_field :owner_secret, value: params[:owner_secret] %>
+      <% needs_forem_owner_secret_field = ENV["FOREM_OWNER_SECRET"].present? && Settings::General.waiting_on_first_user %>
```

Comment:
Could it make sense to wrap this up in our `ForemInstance` model? Something like `ForemInsance.needs_owner_secret?` or something?

Reply:
This is a good idea - thanks Michael.